package.json

```diff
@@ -15,7 +15,8 @@
   "dependencies": {
     "@google-cloud/storage": "^1.7.0",
     "foam2": "git://github.com/foam-framework/foam2.git#mdn-confluence",
-    "web-api-confluence-dashboard": "git://github.com/mdittmer/confluence.git#mdn",
+    "mdn-browser-compat-data": "git://github.com/mdn/browser-compat-data.git",
```

Comment:
We need to keep this as a peer dependency to avoid a dependency cycle when BCD embeds us.